src/tests/synthetic-stx-txs-tests.ts

```diff
@@ -0,0 +1,347 @@
+import { ChainID } from '@stacks/transactions';
+import * as fs from 'fs';
+import * as path from 'path';
+import { DecodedTxResult, TxPayloadTypeID } from 'stacks-encoding-native-js';
+import { CoreNodeBlockMessage } from '../event-stream/core-node-message';
+import { parseMessageTransaction } from '../event-stream/reader';
+
+// Test processing of the psuedo-Stacks transactions, i.e. the ones that
+// originate on the Bitcoin chain, and have a `raw_tx == '0x00'.
+
+test('test synthetic tx token transfer 1', () => {
```

Comment:
Can you wrap these tests inside a `describe('synthetic stx txs', () =>`?
So we can filter them out easily on Jest in the future

Reply:
Done!